examples/co2_and_tracer_analysis.py

```diff
@@ -16,7 +16,7 @@
 # (which can be created the workflow presented in the Jupyter notebook
 # examples/notebooks/curvature_correction_walkthrough.ipynb).
 curvature_correction = daria.CurvatureCorrection(
-    config_source=image_folder + "config.json", width=2.8, height=1.5
+    config=image_folder + "config.json", width=2.8, height=1.5
```

Comment:
```suggestion
    config=image_folder + "config.json"
```
Already included in the config file.